docs/partner_editable/additional_info.adoc

```diff
@@ -21,15 +21,6 @@ The Application Load Balancer handles redirection based on the host header in re
 
 For guidance on this process, see your DNS provider's documentation.
 
-=== Create Amazon Cognito user identities
-
-The Application Load Balancer defines an authentication action to protect the AIT, Open MCT, and AIT Sequence Editor applications. To access the applications, you authenticate using an identity that's managed by Amazon Cognito. Create user identities in the Amazon Cognito user pool or set up federation with an existing identity provider.
-
-For more information, see the following:
-
-* https://docs.aws.amazon.com/cognito/latest/developerguide/how-to-create-user-accounts.html[Creating User Accounts as Administrator^]
-* https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-identity-federation.html[Adding User Pool Sign-in Through a Third Party^]
-
```

Comment:
We definitely still want to keep this in the docs - this tells a user how to set up Cognito users or identity federation for their deployment. What we have added here is a single, default user so that they can test as soon as the stack comes up. They will still need to set up identities for long-term use

Reply:
Sweet added back in!